### PR TITLE
Move structure tests out of getting-started example

### LINKED ISSUE #1154: Split getting-started into two examples
I think the getting-started should be the smallest possible example.

current
```yaml
apiVersion: skaffold/v1alpha4
kind: Config
build:
  artifacts:
  - image: gcr.io/k8s-skaffold/skaffold-example
test:
  - image: gcr.io/k8s-skaffold/skaffold-example
    structureTests:
      - ./test/*
deploy:
  kubectl:
    manifests:
      - k8s-*
profiles:
  - name: gcb
    build:
      googleCloudBuild:
        projectId: k8s-skaffold
  - name: test
    test:
      - image: gcr.io/k8s-skaffold/skaffold-example
        structureTests:
          - ./test/profile_structure_test.yaml
```

proposed
```yaml
apiVersion: skaffold/v1alpha4
kind: Config
build:
  artifacts:
  - image: gcr.io/k8s-skaffold/skaffold-example
deploy:
  kubectl:
    manifests:
```

### PR DESCRIPTION
**Pull request by nkubala**
Fixes #1154 and #1157

### CODE REVIEW

File: integration/examples/bazel/bazel-genfiles [hunk 1]
```diff
@@ -0,0 +1 @@
+/usr/local/google/home/nkubala/.cache/bazel/_bazel_nkubala/0b5f2432d049b24dd37a3f57ec8ef151/execroot/skaffold/bazel-out/k8-fastbuild/genfiles
```
Comment (r2d4) [inline integration/examples/bazel/bazel-genfiles]: integration/examples/bazel/bazel-* should be added to the gitignore and not checked in

Comment (balopat) [reply integration/examples/bazel/bazel-genfiles]: thanks for catching that!! I'll remove those.

Comment (balopat) [reply integration/examples/bazel/bazel-genfiles]: https://github.com/GoogleContainerTools/skaffold/pull/1229/files

Comment (nkubala) [reply integration/examples/bazel/bazel-genfiles]: 😨 thanks @r2d4

**Pull request merged**